core-api/src/main/java/com/optimizely/ab/notification/NotificationCenter.java

```diff
@@ -33,13 +33,32 @@
  * more flexible.
  */
 public class NotificationCenter {
+    public enum OnDecisionNotificationType {
+        IS_FEATURE_ENABLED("feature"),
```

Comment:
This should be `DecisionNotificationType` and suggest changing:
IS_FEATURE_ENABLED --> FEATURE
GET_FEATURE_VARIABLE --> FEATURE_VARIABLE